homeassistant/components/media_player/plex.py

```diff
@@ -145,6 +145,10 @@ def update_devices():
             _LOGGER.error("Could not connect to plex server at http://%s",
                           host)
             return
+        except requests.exceptions.ConnectionError:
+            _LOGGER.error("Could not connect to plex server at http://%s",
+                          host)
```

Comment:
You might want to consider catching RequestException, see http://docs.python-requests.org/en/latest/user/quickstart/#errors-and-exceptions .

Other than that, I'd recommend using except .. as:
```
except X as ex:
    _LOGGER.error("asdf: %s" % ex)
```
to give a hint to the user about what type of error happened.